react/images/InlineGrid.js

```diff
@@ -1,20 +1,21 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import Use from 'vtex.react-use-svg/Use'
 
 export default function InlineGrid({ active }) {
   const color = active ? '#828282' : '#ececec'
```

Comment:
You can use Tachyons here if add a `div` parent and set its color by using tachyons' classes.